src/wovnio/wovnphp/Store.php

```diff
@@ -79,6 +79,8 @@ private function defaultSettings() {
         'wovn_dev_mode' => false,
         'use_server_error_settings' => false,
         'disable_api_request_for_default_lang' => false,
+        'ignore_paths' => array(),
+        'ignore_globs' => array(),
```

Comment:
Down in the code, it is using `preg_match`, so should we rename this setting `ignore_regex` instead?

Reply:
we use ignore globs on wovnrb so I thought it made sense to use globs here

Reply:
I agree we should have unified settings across backends, but if we use this setting as a regular expression user may input actual GLOB pattern that won't work.

Reply:
ok I will use regex you are right

Reply:
thanks